services/minespace-web/src/styles/components/ProjectSummaries.scss

```diff
@@ -0,0 +1,3 @@
+.fixed-header {
+  // position: fixed;
+}
```

Comment:
this can be deleted, my bad.

Reply:
I got rid of this in a previous PR so not sure how it got here, but will remove